decidim-admin/spec/commands/decidim/admin/participatory_space/publish_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Any Participatory Space can be published", type: :system do
+  it_behaves_like "Publicable space", :participatory_process
```

Comment:
Just thinking out loud, but we don't have a `dummy_participatory_space`  or something like that, right? I couldn't find it.

Reply:
Maybe is time to have ? I see some modules are being connected just to make tests work